### PR TITLE
Update mdn_plus_privacy.md

### PR DESCRIPTION
**Pull request by aroybalreid**
Correcting the "last updated" date to reflect recent changes

### CODE REVIEW

**Review by aroybalreid — COMMENTED** [review]
I forgot to update the year (should be 2023).  Can we update that as well, or do I need to submit a new request?

File: en/mdn_plus_privacy.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## <span class="privacy-header-firefox">MDN Plus</span> <span class="privacy-header-policy">Privacy Notice</span>
 
-Version 1.0, Effective March 23, 2022
+Version 1.0, Effective May 24, 2022
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:3]: need a new version number too.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Would you also include the change of the same items in the ToS doc too? Afterwards, I will send them out to the vendor.

**Comment by peiying2** [comment]
@aroybalreid would you please also fix the ToS date/version too 

> I forgot to update the year (should be 2023). Can we update that as well, or do I need to submit a new request?

You can make another change to the same PR. Make sure to change the version number too.

**Comment by flodolo** [comment]
You also updated en/mdn_plus_privacy.md. Safe to assume that should be updated in the same way?

**Review by flodolo — APPROVED** [review]

**Pull request merged**